### PR TITLE
Additional logs

### PR DESCRIPTION
**Pull request by noescape00**

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.BlockStore/Controllers/BlockStoreController.cs [hunk 1]
```diff
@@ -151,11 +151,15 @@ public IActionResult GetAddressesBalances(string addresses, int minConfirmations
             {
                 string[] addressesArray = addresses.Split(',');
 
+                this.logger.LogDebug($"Asking data for {addressesArray.Length} addressees.");
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.BlockStore/Controllers/BlockStoreController.cs]:
```suggestion
                this.logger.LogDebug($"Asking data for {addressesArray.Length} addresses.");
```

File: src/Stratis.Bitcoin/Controllers/RestApiClientBase.cs [hunk 2]
```diff
@@ -106,6 +106,8 @@ public RestApiClientBase(ILoggerFactory loggerFactory, IHttpClientFactory httpCl
         public async Task<Response> SendGetRequestAsync<Response>(string apiMethodName, string arguments = null,
             CancellationToken cancellation = default(CancellationToken)) where Response : class
         {
+            this.logger.LogTrace("{0}:{1}, {2}:{3}", nameof(apiMethodName), apiMethodName, nameof(arguments), arguments);
```
Comment (noescape00) [inline src/Stratis.Bitcoin/Controllers/RestApiClientBase.cs]: we already have this with fody enabled. No need to add it manually.

File: src/Stratis.Features.FederatedPeg/CollateralChecker.cs [hunk 3]
```diff
@@ -77,6 +77,7 @@ public async Task InitializeAsync()
             foreach (CollateralFederationMember federationMember in this.federationManager.GetFederationMembers()
                 .Cast<CollateralFederationMember>().Where(x => x.CollateralAmount != null && x.CollateralAmount > 0))
             {
+                this.logger.LogTrace("Initializing federation member {0} with amount {1}.", federationMember.CollateralMainchainAddress, federationMember.CollateralAmount);
```
Comment (noescape00) [inline src/Stratis.Features.FederatedPeg/CollateralChecker.cs]: All non-entry and non-exit logs should bee on debug level

File: src/Stratis.Bitcoin/Controllers/RestApiClientBase.cs (hunk 2 above)
Comment (fassadlr) [reply src/Stratis.Bitcoin/Controllers/RestApiClientBase.cs]: OK yeah will remove!

File: src/Stratis.Features.FederatedPeg/CollateralChecker.cs (hunk 3 above)
Comment (fassadlr) [reply src/Stratis.Features.FederatedPeg/CollateralChecker.cs]: Will Change

**Comment by noescape00** [comment]
@fassadlr please reapprove

**Pull request merged**